components/03-modules/side-menu/SideMenu.scss

```diff
@@ -0,0 +1,109 @@
+$side-menu__item-hover-bg-color     : #f6f6fa !default;
```

Comment:
is this color defined in globals?

Reply:
Unfortunately is not. Should I move it into globals?